packages/client/src/components/SafeSettings.js

```diff
@@ -447,11 +448,29 @@ function SafeSettings({ address, web3, name, threshold, safeOwners }) {
     const thresholdToPersist = newThreshold ?? threshold;
 
     if (newOwner) {
-      await proposeAddSigner(formatAddress(newOwner.address));
+      const intent = `Add account ${newOwner.address} as a signer.`;
+      const { message, keyIds, signatures, height } = createSignature(web3, intent);
```

Comment:
Instead of defining params here and pass through multiple methods to reach the one in `useTreasury.js`, can we consider define the params inside `useTreasury.js` itself?

Reply:
Makes sense, will update it! :)

Reply:
Updated, please take a look again when you have time